client/extensions/woocommerce/app/settings/email/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import ActionHeader from 'woocommerce/components/action-header';
+import Button from 'components/button';
+import SettingsNavigation from '../navigation';
+import { getLink } from 'woocommerce/lib/nav-utils';
```

Comment:
This should be moved above `SettingsNavigation` to keep things in alphabetical order.